rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -536,7 +538,7 @@ public synchronized Ethereum getRsk() {
         return rsk;
     }
 
-    private GasPriceTracker getGasPriceTracker() {
+    public GasPriceTracker getGasPriceTracker() {
```

Comment:
```suggestion
    public GasPriceTracker getGasPriceTracker() {
        checkIfNotClosed();
        
```

the test doesn't fail because `getBlockStore()` has this check, but I'd rather perform this check explicitly

Reply:
Perfect! I had a doubt about that too, because it was somewhat "hidden" also, the faster it fails, the better. Will commit your suggestion. Thanks!